python-threatexchange/threatexchange/extensions/vpdq/vpdq_index.py

```diff
@@ -29,6 +31,8 @@ class VPDQIndex(SignalTypeIndex[IndexT]):
     def __init__(
         self,
         quality_threshold: int = VPDQ_QUALITY_THRESHOLD,
+        query_match_threshold: float = VPDQ_QUERY_MATCH_THRESHOLD,
+        index_match_threshold: float = VPDQ_INDEX_MATCH_THRESHOLD,
```

Comment:
nit: Would strongly recommend using `_pct` or `_percent` here, as well as asserting it's 0 <= val <= 100 

You may also want to name the variable such that it's clear it's a lowerbound - for all other signal types, we use % difference (so 0 = exact match, 100 = accept anything). With vPDQ it's reversed.